### DISCUSSION TITLE
Backend tokens ?

### DISCUSSION THREAD

**Discussion by leElvyn** [pocketbase/pocketbase#838] [Q&A] [answered]
Hi, i have a simple svelte kit app, and i would like some actions to only be performed from the backend. Is it possible to generate in the admin UI tokens for admins, without having to "expose" to the backend that admin's email / password ?

**Comment by ganigeorgiev** [comment, marked as answer]
No, currently there is no support for "API key"-like tokens. You can create a new system admin account and use that in your ssr action to authenticate.

**Comment by leElvyn** [comment]
Ok, thanks. This would make for a nice addition for pocketbase !
